src/bot/commands/embedsay.js

```diff
@@ -0,0 +1,52 @@
+// Says the contents in the specified channel inside an embed, or in the same channel if no channel is passed
+
+import {Command} from 'yuuko';
+
+const command = new Command('embedsay', async (msg, args, context) => {
+	if (!args.length) {
+		return context.sendHelp(msg, context);
+	}
+
+	// try to get the channel, if none is found just post it in the channel the message was sent
+	let channel;
+	try {
+		channel = context.client.getChannel(args[0].split('\n')[0]) || await context.client.getRESTChannel(args[0]);
+	} catch (_) {
+		// pass, channel stays undefined
+	}
+
+	if (channel) {
+		const messageContent = args.slice(1, args.length).join(' ');
+		// The character limit for embed descriptions is 4096 so I set 4000 to leave room for the IDs though 96 might be overkill
+		if (messageContent.length > 4000) {
+			msg.channel.createMessage('Message is too big!').catch(() => {});
+			return;
+		}
+
+		context.client.createMessage(channel.id, {
+			embed: {
+				description: messageContent,
+			},
+		}).catch(() => {});
+	} else {
+		const messageContent = args.slice(0, args.length).join(' ');
```

Comment:
This `slice()` call does nothing, I'm pretty sure? It returns a clone of the full array, but we don't need a clone to `join()`, we can do that directly on the original array

```suggestion
		const messageContent = args.join(' ');
```

Reply:
Yep removed it